resources/sdk/purecloudpython/extensions/PureCloudRegionHosts.py

```diff
@@ -0,0 +1,18 @@
+from enum import Enum
+
+
+class PureCloudRegionHosts(Enum):
```

Comment:
The name of this class is good, but the filename should be `purecloud_region_hosts.py` to be consistent with the generated files.